### ISSUE TITLE
Release Notes feedback: Structure of chapters 2 and 3

### ISSUE THREAD

**Issue by ccopelloRH** [fabric8-launcher/launcher-documentation#945]
Per Ladislav's feedback:
I don't quite understand why we split the content of chapters 2 and 3. Actually the entire support situation is a mess from my perspective, and clarity of the release notes suffers (a LOT).

**Comment by inoxx03** [comment]
Thank you for the feedback:) I think we've discussed this somewhere, but the reason the chapters are split the way they are, is because they each cover a separate supportability concept:

Chapter 2 Deals with 3rd party components (that is, unsupported frameworks that can be used with RHOAR) RHOAR has a complicated support situation which is outlined in the [Supported Configurations document](https://docs.google.com/document/d/1nY9Ku_oUWFHj-PDdTajhjMj1DWidswbsUb7fAlcYBgY/edit).

Chapter 3 covers the supportability limitations for components provided as Technology Preview. The tech preview status is [well-defined](https://access.redhat.com/support/offerings/techpreview) in the context of Red Hat products.

The reason why these concepts are relegated to separate chapters is to avoid mixing them.

TL;DR: Chapters 2  and 3 are split, because, Tech preview components =/= 3rd party components in terms of supportability limitations.

**Comment by Ladicek** [comment]
Aha, I see! It probably confused me, because I'd expect a completely different ordering: first describe what's "fully supported", then "tech preview" and other kinds of "limited support", and last "3rd party frameworks".

But I agree now that the split makes sense :-)

**Comment by inoxx03** [comment]
I see your point. The ordering is not optimal here.  And I'd pretty much be in favor of the ordering you mentioned (i.e. from _Fully Supported_  to _3rd Party_) 
We've originally intended to have a support statement saying what's fully supported, but John requested we remove that as it was out of date.

The specifics of the different support tiers for  3rd party frameworks is included in the Supported configurations doc. I figured it might not be a good idea to reduplicate that content in the RN (especially since the SupConf document hasn't (to my knowledge) been published yet and could change anytime).

So, to minimize the impact of potential changes, I linked to the document (link is still a placeholder until I get a definite on the publishing location).

I used Chapter 2 and 3 as disclaimers pointing out what is NOT supported  and then Marked the sections listing anything that's supported as _Supported_.

**Comment by inoxx03** [comment]
@Ladicek I suppose we could swap the order of chapters 2 and 3 and include the list of 3rd Party frameworks available with SPring boot in the spring boot section. That would make the information more complete, but I'd be inclined to wait until the list of the 3rd party frameworks is published so the information included therein is verified.

**Comment by Ladicek** [comment]
Yea agree, no huge changes at this point :-)

**Comment by inoxx03** [comment]
The supported configurations have been incorporated into the RN book.
It would be feasible to remove Chapter 2 entirely as long as we retain Chapter 6 that contains the Supported Configurations.
I have created an [issue](https://gitlab.cee.redhat.com/red-hat-openshift-application-runtimes-documentation/rhoar-release-notes/issues/49) in the Release Notes repo to cover the removal of Chapter 2.

I propose closing this issue now, since we are in the process of decomissioning GH issues in favor of JIRA for RHOAR docs.
